backend/src/main/java/com/woowacourse/kkogkkog/coupon/presentation/UnregisteredCouponController.java

```diff
@@ -0,0 +1,67 @@
+package com.woowacourse.kkogkkog.coupon.presentation;
+
+import com.woowacourse.kkogkkog.common.presentation.LoginMemberId;
+import com.woowacourse.kkogkkog.coupon.application.UnregisteredCouponService;
+import com.woowacourse.kkogkkog.coupon.application.dto.UnregisteredCouponDetailResponse;
+import com.woowacourse.kkogkkog.coupon.application.dto.UnregisteredCouponResponse;
+import com.woowacourse.kkogkkog.coupon.presentation.dto.UnregisteredCouponCreateRequest;
+import com.woowacourse.kkogkkog.coupon.presentation.dto.UnregisteredCouponsResponse;
+import java.util.List;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/api/v2/coupons/unregistered")
+public class UnregisteredCouponController {
+
+    private final UnregisteredCouponService unregisteredCouponService;
+
+    public UnregisteredCouponController(UnregisteredCouponService unregisteredCouponService) {
+        this.unregisteredCouponService = unregisteredCouponService;
+    }
+
+    @GetMapping
+    public ResponseEntity<UnregisteredCouponsResponse> showAll(@LoginMemberId Long loginMemberId) {
+        List<UnregisteredCouponResponse> unregisteredCoupons = unregisteredCouponService.findAllBySender(
+            loginMemberId);
+        return ResponseEntity.ok(new UnregisteredCouponsResponse(unregisteredCoupons));
```

Comment:
DTO명이 너무 긴데 변수추론을 쓰는게 어떨까요? 파일 위쪽의 import문의 개수도 많이 줄 것 같아요

```suggestion
    public ResponseEntity<UnregisteredCouponsResponse> showAll(@LoginMemberId Long loginMemberId) {
        final var unregisteredCoupons = unregisteredCouponService.findAllBySender(loginMemberId);
        return ResponseEntity.ok(new UnregisteredCouponsResponse(unregisteredCoupons));
```

Reply:
좋은 제안인 것 같습니다. 미등록 쿠폰 dto에 한해 var를 사용하려 하는데 다른 분들은 어떻게 생각하시나요? 모두 괜찮으면 적용하겠습니다.